ui/src/app/sensors/components/sensor-details/sensor-details.tsx

```diff
@@ -96,6 +117,16 @@ export const SensorDetails = ({match, location, history}: RouteComponentProps<an
                 <ErrorNotice error={error} />
                 {!sensor ? <Loading /> : <SensorEditor sensor={sensor} onChange={setSensor} onError={setError} selectedTabKey={tab} onTabSelected={setTab} />}
             </>
+            {!!selectedLogNode && (
```

Comment:
Should we use the named slide panel pattern that we have on workflows list? Makes it easy to add new panels in the future:

https://github.com/argoproj/argo/blob/339bcebfbcdbe9703111335892897fd48e298882/ui/src/app/workflows/components/workflows-list/workflows-list.tsx#L184-L189